app/controllers/projects/contributions_controller.rb

```diff
@@ -7,9 +7,11 @@ class Projects::ContributionsController < ApplicationController
   #has_scope :page, default: 1
   after_filter :verify_authorized, except: [:index]
   belongs_to :project
-  before_filter :detect_old_browsers, only: [:new, :create]
-  before_filter :load_channel, only: [:edit, :new]
+  before_action :detect_old_browsers, only: [:new, :create]
```

Comment:
@carloslopes this method is only used by the ProjectsController. The other controllers not use it, so I don't know if this change need to be made now.